src/components/Garden/Feed/WrapToken.js

```diff
@@ -54,21 +40,32 @@ function WrapToken({ onClaimRewards, onUnwrapToken, onWrapToken }) {
     <Token
       balance={wrappableToken.accountBalance}
       loading={loading}
-      mode="wrap"
+      mode={{
+        icon: unwrappedIcon, 
+        button: { mode: 'strong', label: 'Wrap' },
+        apy: rewardAPY
+      }}
```

Comment:
let's use the values that we just got: 

```suggestion
      mode={{
        icon: unwrappedImage,
        button: { mode: 'strong', label: 'Wrap' },
        apy: rewardAPYFormatted
      }}
```